test/unit/server-psp.bats

```diff
@@ -0,0 +1,315 @@
+      . | tee /dev/stderr |
+      yq -r '.metadata.annotations["vault-is"]' | tee /dev/stderr)
+  [ "${actual}" = "amazing" ]
+
+  local actual=$(helm template \
+      --show-only templates/server-psp.yaml  \
+      --set 'server.ha.enabled=true' \
+      --set 'global.psp.enable=true' \
+      --set 'global.psp.annotations.vault-is=amazing' \
+      . | tee /dev/stderr |
+      yq -r '.metadata.annotations["vault-is"]' | tee /dev/stderr)
+  [ "${actual}" = "amazing" ]
+
+  local actual=$(helm template \
+      --show-only templates/server-psp.yaml  \
+      --set 'server.standalone.enabled=true' \
+      --set 'global.psp.enable=true' \
+      --set 'global.psp.annotations.vault-is=amazing' \
+      . | tee /dev/stderr |
+      yq -r '.metadata.annotations["vault-is"]' | tee /dev/stderr)
+  [ "${actual}" = "amazing" ]
+}
+
+@test "server/PodSecurityPolicy: disable with global.enabled false" {
+  cd `chart_dir`
+  local actual=$( (helm template \
+      --show-only templates/server-psp.yaml  \
+      --set 'server.dev.enabled=true' \
+      --set 'global.enabled=false' \
+      --set 'global.psp.enable=true' \
+      . || echo "---") | tee /dev/stderr |
+      yq 'length > 0' | tee /dev/stderr)
+  [ "${actual}" = "false" ]
+
+  local actual=$( (helm template \
+      --show-only templates/server-psp.yaml  \
+      --set 'server.ha.enabled=true' \
+      --set 'global.enabled=false' \
+      --set 'global.psp.enable=true' \
+      . || echo "---") | tee /dev/stderr |
+      yq 'length > 0' | tee /dev/stderr)
+  [ "${actual}" = "false" ]
+
+  local actual=$( (helm template \
+      --show-only templates/server-psp.yaml  \
+      --set 'server.standalone.enabled=true' \
+      --set 'global.enabled=false' \
+      --set 'global.psp.enable=true' \
+      . || echo "---") | tee /dev/stderr |
+      yq 'length > 0' | tee /dev/stderr)
+  [ "${actual}" = "false" ]
+}
+
+@test "server/PodSecurityPolicy: disable with global.psp.enable false" {
+  cd `chart_dir`
+  local actual=$( (helm template \
+      --show-only templates/server-psp.yaml  \
+      --set 'server.dev.enabled=true' \
+      --set 'global.psp.enable=false' \
+      . || echo "---") | tee /dev/stderr |
+      yq 'length > 0' | tee /dev/stderr)
+  [ "${actual}" = "false" ]
+
+  local actual=$( (helm template \
+      --show-only templates/server-psp.yaml  \
+      --set 'server.ha.enabled=true' \
+      --set 'global.psp.enable=false' \
+      . || echo "---") | tee /dev/stderr |
+      yq 'length > 0' | tee /dev/stderr)
+  [ "${actual}" = "false" ]
+
+  local actual=$( (helm template \
+      --show-only templates/server-psp.yaml  \
+      --set 'server.standalone.enabled=true' \
+      --set 'global.psp.enable=false' \
+      . || echo "---") | tee /dev/stderr |
+      yq 'length > 0' | tee /dev/stderr)
+  [ "${actual}" = "false" ]
+}
+
+@test "server/PodSecurityPolicy: disable with global.enabled false global.psp.enable.enabled false" {
```

Comment:
The test parameters don't match this line? Is global.psp.enable supposed to be true or false?

Reply:
Deleted these. These are the same tests as lines 173-198